### ISSUE TITLE
Edits made to the different section fo the user profile are not represented in the profile are the changes are made. 

### ISSUE THREAD

**Issue by bckingsbury** [topcoderinc/mjolnir-web#126]
### OS and Browser Details ###
*iPad Pro - 11.2.6
### Screen Name where the issue occurs ###
*edit User profile 
### Exact Reproduction Steps ###
1. Make changes to any of the sections of the user profile that bring up a pop up 
2. 

### Actual  Result ###
See attached

### Expected Result ###
See attached

### Screenshots/Video ###
*Paste screenshots here. For video, compress and attach*
![c09d2474-74a3-400d-8e4a-ebaa5eb3242f](https://user-images.githubusercontent.com/31655780/36699204-005093c0-1b1a-11e8-9676-39e2cb8497c1.jpeg)
![dabd4a3d-99da-4098-b86e-19b14c69d37e](https://user-images.githubusercontent.com/31655780/36699205-00663ad6-1b1a-11e8-8a39-e2c5f0bf823a.jpeg)

**Comment by mahestro** [comment]
True, green ticks should be there according to the design https://drive.google.com/file/d/1XeGV83Ktoi0j0Z2O7QAgFNhwVZvjF_UI/view?usp=sharing
